src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Startup.cs

```diff
@@ -58,44 +59,45 @@ public void ConfigureServices(IServiceCollection services)
 #if (IndividualLocalAuth)
             services.AddDbContext<IdentityDbContext>(options =>
     #if (UseLocalDB)
-                options.UseSqlServer(
-                    Configuration.GetConnectionString("DefaultConnection"),
-                    sqlOptions => sqlOptions.MigrationsAssembly("Company.WebApplication1")
-                ));
+                    options.UseSqlServer(
+                        Configuration.GetConnectionString("DefaultConnection"),
+                        sqlOptions => sqlOptions.MigrationsAssembly("Company.WebApplication1")
```

Comment:
Is this a tab? WHat's up with these whitepsace changes?

Reply:
I'll fix those